### ISSUE TITLE
Settings Section Topmost Level 2022

### ISSUE THREAD

**Issue by johnagl** [phynet/iOS-URL-Schemes#13]
`Apple will reject apps that are using private url schemes (Ugh, Apple....) if they are pretty much obvius. Some apps are rejected and others are not, so, be aware of this issue before implementing any of those URL's in your app as a feature.`

Is there still no way to navigate to the settings app without getting rejected by the Apple Store?

Currently in development  (ios 15.4) I am able to navigate to the settings section using `App-prefs:`.

**Comment by phynet** [comment]
Not that I'm aware of. But you still can try to apply it and check if you pass the review 🤷‍♀️  Good luck!

**Comment by johnagl** [comment]
> Not that I'm aware of. But you still can try to apply it and check if you pass the review 🤷‍♀️ Good luck!

Yep, will submit for review soon ... there are people who say navigating to the top level of settings is allowed. I'll respond here with the results.

**Comment by phynet** [comment]
Did you submit it? did it work?

**Comment by johnagl** [comment]
@phynet It worked - got approved.

**Comment by phynet** [comment]
Cool. Let's hope it remains like that. 😌
